modules/distribution/src/repository/resources/dbscripts/apimgt/oracle.sql

```diff
@@ -321,7 +323,7 @@ ALTER TABLE SP_INBOUND_AUTH ADD CONSTRAINT APPLICATION_ID_CONSTRAINT FOREIGN KEY
 CREATE TABLE SP_AUTH_STEP (
             ID INTEGER,
             TENANT_ID INTEGER NOT NULL,
-            STEP_ORDER INTEGER DEFAULT 1,
+	     	STEP_ORDER INTEGER DEFAULT 1,
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04